### PR TITLE
Update Readme: Releases

### PR DESCRIPTION
**Pull request by Rasmizzle**
Adds basic release sequence for devs

Feel free to complete/correct this

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ Define variables in your `package.json`:
 }
 ```
 
+> alternatively you can use keys `BASE_PACKAGE_NAME` and `SITE_PACKAGE_NAME` in an .env file in the directory of your package.json. This is mandatory when using workspaces in a monorepo as package-variables are not always available in the workspace.
```
Comment (hlubek) [inline README.md:27]: Sentence should start uppercased

s/use key/set the variables/

File: README.md [hunk 2]
```diff
@@ -240,3 +243,21 @@ describe('ComponentToTest', () => {
   }
 }
 ```
+
+
+## Release cycle:
```
Comment (hlubek) [inline README.md:248]: This is more about "how to release" and less the "release cycle", which would be more about when and how often releases are published.

File: README.md [hunk 3]
```diff
@@ -240,3 +243,21 @@ describe('ComponentToTest', () => {
...
+
+### Pre-releases
+
+To create a pre-release one can push/merge changes to branch next. This triggers actions to automatically create a pre-release.
+you can use the current pre-release by using `@next` as version in your project package.json.
```
Comment (hlubek) [inline README.md:253]: Start of sentence should be uppercased

File: README.md [hunk 4]
```diff
@@ -240,3 +243,21 @@ describe('ComponentToTest', () => {
...
+
+### Releases
+
+1. Merge your branch/changes into main-branch
```
Comment (hlubek) [inline README.md:257]: s/main-branch/main branch/

File: README.md [hunk 5]
```diff
@@ -240,3 +243,21 @@ describe('ComponentToTest', () => {
...
+2. bump version in package.json with `npm version [<newversion> | major | minor | patch | premajor | preminor | prepatch | prerelease [--preid=<prerelease-id>]`
```
Comment (hlubek) [inline README.md:258]: * Start of list items should be all uppercase
* I would not include all esoteric arguments here, just `npm version [<newversion> | major | minor | patch`

File: README.md [hunk 6]
```diff
@@ -240,3 +243,21 @@ describe('ComponentToTest', () => {
...
+3. push bumped version to main-branch with `git push --tags`
+4. create a new release with release notes from newly created tag on github
+5. The new release will trigger github-actions for automatically publishing to npm
```
Comment (hlubek) [inline README.md:261]: s/github-actions/GitHub Actions/
s/npm/NPM/

File: README.md [hunk 7]
```diff
@@ -240,3 +243,21 @@ describe('ComponentToTest', () => {
...
+
+TODO: We should invest some time to optimize release handling
```
Comment (hlubek) [inline README.md:263]: This shouldn't be part of the README, let's create an issue for it. But like discussed today: what should be the goal? If the current way is documented and works - it should be okay. Let's rather invest in propert end-to-end tests for upgrades and refactorings.

**Review by hlubek — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Rasmizzle) [reply README.md]: neu: 
## Release a new version:

File: README.md (hunk 4 above)
Comment (Rasmizzle) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (Rasmizzle) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (Rasmizzle) [reply README.md]: done

File: README.md (hunk 7 above)
Comment (Rasmizzle) [reply README.md]: Got it. Created an unrefined issue for end to end tests

**Review by hlubek — APPROVED** [review]

**Pull request merged**